cmd/restic/cmd_diff.go

```diff
@@ -273,6 +274,14 @@ func (c *Comparer) diffTree(ctx context.Context, stats *DiffStatsContainer, pref
 				!reflect.DeepEqual(node1.Content, node2.Content) {
 				mod += "M"
 				stats.ChangedFiles++
+
+				node1NilContent := node1
+				node2NilContent := node2
+				node1NilContent.Content = nil
+				node2NilContent.Content = nil
+				if node1NilContent.Equals(*node2NilContent) {
```

Comment:
This won't work for snapshots in which files have unstable inodes. As the rest of the `diff` command also has that problem, I'm ok with it. But we should add a bit of documentation, that the detection may not work if `backup --ignore-inode` or `--ignore-ctime` were used.